.github/workflows/Basic.yml

```
@@ -1,11 +1,14 @@
 # Based on https://github.com/actions-rs/example/blob/master/.github/workflows/quickstart.yml
 
-on: [push, pull_request]
+on:
+  pull_request:
```

```suggestion
  pull_request:
    - master
```

I seem to remember that you need this

---

I'm not sure to see why would we want to restrict actions to run only for PR targeting master? Running for any PR seems fine?